frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx

```diff
@@ -90,8 +91,8 @@ const RawCapacityCard: React.FC = React.memo(() => {
             </div>
           </>
         )}
-        {!loadError && loading && <LoadingCardBody />}
-        {loadError && !loading && <ErrorCardBody />}
+        {loading && !loadError && <LoadingCardBody />}
```

Comment:
In case of all undefined values can't we should not available?

Reply:
In case of all `undefined` values `loadError` will be `true` and this block will not run. Line 95 with `ErrorCardBody` having `Not Available` will get activated.

Reply:
righto!